cmd/minikube/cmd/start_flags.go

```diff
@@ -311,7 +311,7 @@ func generateClusterConfig(cmd *cobra.Command, existing *config.ClusterConfig, k
 			DockerOpt:               config.DockerOpt,
 			InsecureRegistry:        insecureRegistry,
 			RegistryMirror:          registryMirror,
-			HostOnlyCIDR:            viper.GetString(hostOnlyCIDR),
+			HostOnlyCIDR:            strings.Trim(viper.GetString(hostOnlyCIDR), "'"))
```

Comment:
minikube isn't building because you need to add a comma after the last parentheses here

```go
HostOnlyCIDR:            strings.Trim(viper.GetString(hostOnlyCIDR), "'")),
```

Before you push the change, you can verify that the code compiles by running `make`